test/dom/attr.test.js

```diff
@@ -1,71 +1,61 @@
 'use strict'
+const { DOMParser } = require('../../lib/dom-parser')
+const { DOMException } = require('../../lib/dom')
 
-var DOMParser = require('../../lib/dom-parser').DOMParser
-const assert = require('../assert')
-
-// Create a Test Suite
 describe('XML attrs', () => {
 	it('can properly set attribute', () => {
-		var root = new DOMParser().parseFromString('<xml/>', 'text/xml')
+		const root = new DOMParser().parseFromString('<xml/>', 'text/xml')
 			.documentElement
```

Comment:
I would reformat it like this:

```suggestion
		const root = new DOMParser()
			.parseFromString('<xml/>', 'text/xml')
			.documentElement
```

but Prettier does not give us the option: https://github.com/prettier/prettier/issues/7884 ... argh!